src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.symbols.nuspec

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@pranavkm for the first pass (just to fix https://github.com/aspnet/Razor/issues/2215) can we just nuke this and include symbols in the regular NuspecFile? I noticed `dotnet pack` does the right thing. It automatically filters out pdbs from the main nupkg even if symbols are included in the nuspec file.
We can later add the symbols.nuspec file after we are ready to merge aspnet/BuildTools#630.